### PR TITLE
libgettext 0.20.1

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **libgettext/0.20.1**

This recipe comes from https://github.com/bincrafters/conan-gettext

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
[.gitignore ](https://github.com/conan-io/conan-center-index/blob/master/.gitignore#L55)prevents from commiting translation files.  I can reduce the test to reading the library version (`extern LIBINTL_DLL_EXPORTED int libintl_version;`), what do you think ?

**Comment by ericLemanissier** [comment]
also, this contains only the library. Should I rename this recipe to `libgettext` and make another recipe for the executable named `gettext` ?

**Comment by uilianries** [comment]
> also, this contains only the library. Should I rename this recipe to `libgettext` and make another recipe for the executable named `gettext` ?

Yes, please, rename this project to libgettext. The gettext should be only the installer.

**Comment by ericLemanissier** [comment]
@uilianries done
I also created the installer recipe: https://github.com/conan-io/conan-center-index/pull/835

File: recipes/libgettext/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    def _make_args(self):
+        return ["-C", "intl"]
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        if tools.os_info.is_windows:
+            if "CONAN_BASH_PATH" not in os.environ:
```
Comment (uilianries) [inline recipes/libgettext/all/conanfile.py]:
```suggestion
            if "CONAN_BASH_PATH" not in os.environ and \
               tools.os_info.detect_windows_subsystem() != "msys2":            
```

**Pull request merged**